app/src/main/java/fr/free/nrw/commons/achievements/AchievementsActivity.java

```diff
@@ -388,10 +447,10 @@ private void launchAlert(String title, String message){
     private boolean checkAccount(){
         Account currentAccount = sessionManager.getCurrentAccount();
         if (currentAccount == null) {
-        Timber.d("Current account is null");
-        ViewUtil.showLongToast(this, getResources().getString(R.string.user_not_logged_in));
-        sessionManager.forceLogin(this);
-        return false;
+            Timber.d("Current account is null");
```

Comment:
This changes are valid but irrelevant with this PR, please revert them.

Reply:
Fixed int the next commit